libvast/src/system/active_partition.cpp

```diff
@@ -62,6 +62,129 @@
 
 namespace vast::system {
 
+namespace {
+
+bool should_skip_index_creation(const type& type,
+                                const qualified_record_field& qf,
+                                const std::vector<index_config::rule>& rules) {
+  if (type.attribute("skip").has_value()) {
+    return true;
+  }
```

Comment:
I just noticed we're also fixing sc-35515 with this PR.

Can you add one more integration test that imports anything with the `#skip` attribute, flushes the db, and then exports that event again to ensure this works, and add a separate changelog entry (ie. a new paragraph in the `.md` file) for the bugfix?

Reply:
Will be added soon